OutOfSchool/OutOfSchool.WebApi/Controllers/ChildrenController.cs

```diff
@@ -1,54 +1,166 @@
 ﻿using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
-using OutOfSchool.Services;
 using OutOfSchool.Services.Models;
 using OutOfSchool.WebApi.Models;
-using OutOfSchool.WebApi.Services;
 using OutOfSchool.WebApi.Services.Interfaces;
 using System;
 using System.Collections.Generic;
-using System.Linq;
 using System.Threading.Tasks;
 
 namespace OutOfSchool.WebApi.Controllers
 {
+    /// <summary>
+    /// Controller with CRUD operations for Child entity.
+    /// </summary>
     [ApiController]
     [Route("[controller]/[action]")]
     [Authorize]
     public class ChildrenController : ControllerBase
     {
         private IChildService childService;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ChildrenController"/> class.
+        /// </summary>
+        /// <param name="childService">Service for Child model.</param>
         public ChildrenController(IChildService childService)
         {
             this.childService = childService;
         }
 
+        /// <summary>
+        /// Get all children from database.
+        /// </summary>
+        /// <returns>List of all children.</returns>
         [HttpGet]
-        public async Task<ActionResult<IEnumerable<Child>>> GetChildren()
+        public ActionResult<IEnumerable<Child>> GetChildren()
         {
+            IEnumerable<ChildDTO> children;
+            try
+            {
+                children = this.childService.GetAll();
```

Comment:
return statement might be placed here, no need to introduce a variable 'children' in this case